runtime/fiber.c

```diff
@@ -942,6 +955,20 @@ struct stack_info* caml_alloc_main_stack (uintnat init_wsize)
   return stk;
 }
 
+static void free_stack_memory(struct stack_info* stack)
+{
+#if defined(DEBUG) && defined(STACK_CHECKS_ENABLED)
+  memset(stack, 0x42, (char*)stack->handler - (char*)stack);
+#endif
+#ifdef USE_MMAP_MAP_STACK
```

Comment:
I see that this is consistent with the directives in `alloc_for_stack`, but for more local consistency I'd prefer `#if defined(USE_MMAP_MAP_STACK)`